sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_generated/_patch.py

```diff
@@ -6,9 +6,119 @@
 
 Follow our quickstart for examples: https://aka.ms/azsdk/python/dpcodegen/python/customize
 """
-from typing import List
+from typing import List, Optional, Any, Union
+from azure.core import PipelineClient
+from azure.core.credentials import AzureKeyCredential, TokenCredential
+from azure.core.pipeline import policies
 
-__all__: List[str] = []  # Add all objects you want publicly available to users at this package level
+from azure.appconfiguration._generated import models as _models
+from azure.appconfiguration._generated._azure_app_configuration import (
+    AzureAppConfiguration as AzureAppConfigurationGenerated,
+)
+from azure.appconfiguration._generated._configuration import (
+    AzureAppConfigurationConfiguration as AzureAppConfigurationConfigurationGenerated,
+    VERSION,
+)
+from azure.appconfiguration._generated._serialization import Deserializer, Serializer
+
+
+class AzureAppConfiguration(AzureAppConfigurationGenerated):
+    """AzureAppConfiguration.
+
+    :param credential: Credential needed for the client to connect to Azure. Required.
+    :type credential: ~azure.core.credentials.TokenCredential or ~azure.core.credentials.AzureKeyCredential
+    :param endpoint: The endpoint of the App Configuration instance to send requests to. Required.
+    :type endpoint: str
+    :param sync_token: Used to guarantee real-time consistency between requests. Default value is
+     None.
+    :type sync_token: str
+    :keyword api_version: Api Version. Default value is "2023-10-01". Note that overriding this
+     default value may result in unsupported behavior.
+    :paramtype api_version: str
+    :keyword int polling_interval: Default waiting time between two polls for LRO operations if no
+     Retry-After header is present.
+    """
+
+    def __init__(
```

Comment:
@YalinLi0312 are you just trying to make credential accept both TokenCredential and AzureKeyCredential?

If so, all you need for your customization is the following snippet, you don't need to copy everything over

```python
class AzureAppConfiguration(AzureAppConfigurationGenerated):
  def __init__(
    self,
    credential: Union[TokenCredential, AzureKeyCredential],
    endpoint: str,
    sync_token: Optional[str] = None,
    **kwargs: Any
  ):
    if isinstance(credential, AzureKeyCredential):
      authentication_policy = policy.AzureKeyCredentialPolicy(credential...)
    elif hasattr(self.credential, "get_token"):
      authentication_policy = policy.BearerTokenCredentialPolicy(credential, credential_scopes, **kwargs)
    else:
      raise TypeError(f"Unsupported credential: {credential}")
    super().__init__(credential, endpoint, authentication_policy=authentication_policy, **kwargs)
```

And that's it

Reply:
We prepared the `authentication_policy` in custom layer, call `super().__init__(...)` would be good then.